jspoon/src/main/java/pl/droidsonroids/jspoon/HtmlAdapter.java

```diff
@@ -131,11 +132,11 @@ public T fromInputStream(InputStream inputStream, URL baseUrl) throws IOExceptio
      * @return Created object of type {@code T}
      * @throws IOException If I/O error occurs while reading the {@code InputStream}
      */
-    public T fromInputStream(InputStream inputStream, Charset charset, URL baseUrl, T instance) throws IOException {
+    public T fromInputStream(@NotNull InputStream inputStream, @Nullable Charset charset, @Nullable URL baseUrl, @Nullable T instance) throws IOException {
         String urlToUse = baseUrl != null ? baseUrl.toString() : null;
         String charsetToUse = charset != null ? charset.name() : null;
         Element root = Jsoup.parse(inputStream, charsetToUse, urlToUse);
-        return loadFromNode(root, null);
```

Comment:
This was an oversight - the `instance` parameter was unused, making this method functionally identical to the overload without it.